src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Duke is a program that helps the user manage their tasks.
+ */
+public class Duke {
+    private static final Scanner sc = new Scanner(System.in);
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    enum TaskType {
```

Comment:
I like your use of enums here! Would it also be possible to create a separate enum class so you need not call Duke.TaskType from other classes?